luxonis_train/nodes/heads/discsubnet_head/blocks.py

```diff
@@ -0,0 +1,166 @@
+import torch
+from torch import Tensor, nn
+
+from luxonis_train.nodes.blocks import ConvModule
+
+
+def conv_block(in_channels: int, out_channels: int) -> nn.Sequential:
+    return nn.Sequential(
+        ConvModule(
+            in_channels,
+            out_channels,
+            kernel_size=3,
+            padding=1,
+            activation=nn.ReLU(inplace=True),
+        ),
+        ConvModule(
+            out_channels,
+            out_channels,
+            kernel_size=3,
+            padding=1,
+            activation=nn.ReLU(inplace=True),
+        ),
+    )
+
+
+def upsample_block(in_channels: int, out_channels: int) -> nn.Sequential:
+    return nn.Sequential(
+        nn.Upsample(scale_factor=2, mode="bilinear", align_corners=True),
+        ConvModule(
+            in_channels,
+            out_channels,
+            kernel_size=3,
+            padding=1,
+            activation=nn.ReLU(inplace=True),
+        ),
+    )
+
+
+class Encoder(nn.Module):
+    def __init__(self, in_channels: int, base_width: int) -> None:
+        super(Encoder, self).__init__()
```

Comment:
The `super` arguments are a relic from python 2, please remove them so we're consistent with the rest of the codebase